.fleet/settings.json

```diff
@@ -0,0 +1,5 @@
+{
+    "editor.guides": [],
+    "excluded": [],
+    "toolchains": []
```

Comment:
Looks like a pretty dead config to me, if it did stuff I could see the point of including it.

Reply:
x   x
   o

very ded yeah.. cut it

Reply:
Accidently included it when recursively adding files. Will remove it.